code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -499,48 +501,51 @@
 		arm_clothes = w_uniform
 	if(wear_suit && (wear_suit.body_parts_covered & HANDS) || wear_suit && (wear_suit.body_parts_covered & ARMS))
 		arm_clothes = wear_suit
-	if(arm_clothes)
-		if(!arm_clothes.unacidable)
-			arm_clothes.acid_act(acidpwr, acid_volume_left)
-			acid_volume_left = max(acid_volume_left - acid_decay, 0)
-			update_inv_gloves()
-			update_inv_w_uniform()
-			update_inv_wear_suit()
+
+	if(!bodyzone_hit || bodyzone_hit == "l_arm" || bodyzone_hit == "r_arm")

```

Comment:
while it's not much work going on needlessly, perhaps this if check should be above where we find things such as ```arm_clothes``` and ```chest_clothes```?